php/blocks/controls/class-rich-text.php

```diff
@@ -0,0 +1,78 @@
+<?php
+/**
+ * Rich Text control.
+ *
+ * @package   Block_Lab
+ * @copyright Copyright(c) 2018, Block Lab
+ * @license http://opensource.org/licenses/GPL-2.0 GNU General Public License, version 2 (GPL-2.0)
+ */
+
+namespace Block_Lab\Blocks\Controls;
+
+/**
+ * Class RichText
+ */
+class Rich_Text extends Control_Abstract {
+
+	/**
+	 * Control name.
+	 *
+	 * @var string
+	 */
+	public $name = 'rich_text';
+
+	/**
+	 * Textarea constructor.
+	 *
+	 * @return void
+	 */
+	public function __construct() {
+		parent::__construct();
+		$this->label = __( 'Rich Text', 'block-lab' );
+	}
+
+	/**
+	 * Register settings.
+	 *
+	 * @return void
+	 */
+	public function register_settings() {
+		$this->settings[] = new Control_Setting(
+			array(
+				'name'     => 'help',
+				'label'    => __( 'Help Text', 'block-lab' ),
+				'type'     => 'text',
+				'default'  => '',
+				'sanitize' => 'sanitize_text_field',
+			)
+		);
+		$this->settings[] = new Control_Setting(
```

Comment:
Hmm… what do you think… should we?

a) Remove this option entirely, and force new paragraphs. This is "rich text" and so paragraphs should be expected
b) Enable this by default. I would expect paragraphs in rich text
c) Use the same setting labels / options as we recently implemented for the textarea (doesn't work quite the same though since the textarea isn't an imported control with a `multiline` option)
d) Leave as it (but maybe better align the Help text and Label to the textarea setting?)

Reply:
@RobStino I'd love your thoughts on this, too. I'm leaning toward option A. What's a use case for not automatically creating paragraphs with a rich text field?